openedx/core/djangoapps/user_api/accounts/helpers.py

```diff
@@ -0,0 +1,56 @@
+"""
+Helper functions for the accounts API.
+"""
+import hashlib
+
+from django.conf import settings
+from django.core.files.storage import FileSystemStorage, get_storage_class
+
+PROFILE_IMAGE_SIZES = {
```

Comment:
@cahrens we'll generate all the sizes now, since doing less initially would leave people with an incomplete set later on.  I suppose it's not strictly necessary to give all their URLs from the API now, but not sure if there's any reason not to.